src/frontend/components/UI/OfflineMessage/index.css

```diff
@@ -0,0 +1,36 @@
+.offline-message {
+  background: wheat;
+  padding: 0.5rem;
+  color: black;
+  display: grid;
+  position: relative;
+}
+
+.offline-message span,
+.offline-message button {
+  grid-row: 1;
+  grid-column: 1;
+}
+
+.offline-message button {
+  justify-self: flex-end;
+  align-self: flex-end;
+  background: wheat;
+  border: 2px solid rgb(243, 200, 121);
+}
+
+.offline-message .retry-hint {
```

Comment:
I think adding `text-align: left;` to the retry-hint will make it easier to read

Reply:
hmmm I tried left aligned but it looks weird to me

Reply:
I made it wider, I think it looks better now? maybe the issue is that it's using too many lines? not sure what's best

Reply:
Yeah up to you. It's not much of a difference. I'm fine with either